### DISCUSSION TITLE
PHP repository for Debian 10 (Buster) showing Forbidden

### DISCUSSION THREAD

**Discussion by farrukhahmed4220** [oerdnj/deb.sury.org#2157] [General]
I tried to install the php 8.x & 7.x  but I'm getting Forbidden. I then checked the repository on [https://packages.sury.org/php/dists/buster/] and got 403 Forbidden there.

I understand the EOF of PHP LTS support on Debian 10 but the repository should be there and not removed.
 
Do you have any idea what happened? and how to resolve this?

**Comment by c33s** [comment]
@farrukhahmed4220 have you seen https://github.com/oerdnj/deb.sury.org/issues/2098?

**Reply by farrukhahmed4220** [reply]
Thanks for the reference.

**Reply by jefhar** [reply]
Why does @oerdnj continue to release dead PHP versions, but not dead Debian versions?

**Reply by rfay** [reply]
When he tried to quit on all the old PHP versions we begged him and begged him to continue, and he figured out how.

**Reply by oerdnj** [reply]
@jefhar I am going to assume a genuine question. The answer is that all the PHP versions I provide are fully patched for all the security issues (to some extent, but there’s nothing serious unpatched), while the base system stops being updated after EOL date.

This would be the same as providing packages for unpatched PHP versions full of security holes. And I don’t do that.

Also it’s clearly stated in the policy what I do here. Take it or leave it, just don’t discuss it every time a distribution goes EOL…

**Reply by jefhar** [reply]
@oerdnj thank you, that was a serious question. I thought it a bit hypocritical that you would keep one hunk of EOL packages and discard others. But if you are actually keeping PHP patched, that does make sense. If you can't patch everything, patch one thing.

>Also it’s clearly stated in the policy what I do here

That policy should be displayed in more places if there are discussions every time a distribution goes EOL.

**Reply by waja** [reply]
> That policy should be displayed in more places if there are discussions every time a distribution goes EOL.

Redundancy? Would opt for pointing from those places to the policy. 😀

**Reply by todeveni** [reply]
> That policy should be displayed in more places if there are discussions every time a distribution goes EOL.

Buster EOL announcement is pinned on top of issues page/list and if people would use bug template and actually read the FAQ, which they state they read, it's in there too.

But in todays TikTok age, ain't nobody got time to read.
